tools/trustee/src/cli.rs

```diff
@@ -0,0 +1,171 @@
+use std::path::{Path, PathBuf};
+
+use anyhow::Result;
+use clap::Parser;
+use dirs::home_dir;
+use log::{debug, info, warn};
+use nix::unistd::Uid;
+
+use crate::keys_certs::{
+    ensure_auth_key_pair, ensure_https_cert, ensure_https_key_pair, write_new_auth_key_pair,
+};
+use kbs::attestation::config::AttestationServiceConfig::CoCoASBuiltIn;
+use kbs::{ApiServer, KbsConfig};
+
+fn trustee_keygen(private_path: &Path) -> Result<()> {
+    let public_path = private_path.with_extension("pub");
+    write_new_auth_key_pair(&private_path, &public_path)?;
+    info!("Wrote new private key: {:?}", private_path);
+    info!("Wrote new public key: {:?}", public_path);
+    Ok(())
+}
+
+fn replace_base_dir(path: &Path, new_base: &Path) -> PathBuf {
+    let old_base = "/opt/confidential-containers/";
+    let suffix = if path.starts_with(old_base) {
+        path.strip_prefix(old_base).unwrap()
+    } else {
+        path
+    };
+    new_base.join(suffix)
+}
+
+fn get_config(
+    config_file: Option<PathBuf>,
+    allow_all: bool,
+    trustee_home_dir: &Path,
+) -> Result<KbsConfig> {
+    let mut config = if let Some(path) = config_file {
+        KbsConfig::try_from(path.as_path())?
+    } else {
+        KbsConfig::default()
+    };
+
+    config.policy_engine.policy_path =
+        replace_base_dir(config.policy_engine.policy_path.as_path(), trustee_home_dir);
```

Comment:
Is it guaranteed that the policy path - and other paths passed to `replace_base_dir()` later in this function - will have the base path that `replace_base_dir()` assumes?

Or is it the intended semantics that we redirect specifically and only `/opt/confidential-containers/*` paths to `trustee_home_dir` but leave any other paths alone?  If so I think it would warrant a short explanation in a code comment.